lib/charms/operator_libs_linux/v2/snap.py

```diff
@@ -409,6 +409,25 @@ def unhold(self) -> None:
         """Remove the refresh hold of a snap."""
         self._snap("refresh", ["--unhold"])
 
+    def alias(self, application: str, alias: Optional[str] = None) -> None:
+        """Create an alias for a given application.
+
+        Args:
+            application: application to get an alias.
+            alias: (optional) name of the alias
+        """
+        if alias is None:
+            alias = application
+        _cmd = ["snap", "alias", f"{self.name}.{application}", alias]
```

Comment:
No reason for this variable name to have an underscore prefix; `cmd` is fine. While we're here, in the [subprocess docs for `check_output`](https://docs.python.org/3/library/subprocess.html#subprocess.check_output), this is called `args`, so maybe that's a better name? (It's not just the command.)

I realize the other instances of this use `_cmd`, so maybe consistency is better than being different here. Alternatively, we could replace all the other instances of `_cmd` with `args` too. Up to you.

Reply:
I replaced them in the snap library. Best to have them consistent.